cmd/cli/commands/printers.go

```diff
@@ -532,6 +532,20 @@ func printErrorById(cmd *cobra.Command, errors *pb.ErrorByID) {
 	}
 }
 
+func printStringErrorsById(cmd *cobra.Command, errors *pb.ErrorByStringID) {
```

Comment:
This function is some kind similar to the `printErrorById`. Please, find a way to decompose this.